App/Redux/__tests__/UIRedux.ts

```diff
@@ -0,0 +1,50 @@
+import actions, { reducer } from '../UIRedux'
+
+const initialState = reducer(undefined, {} as any)
+
+describe('ui stories', () => {
```

Comment:
damn writing tests in TS with Jest is a total pleasure!

Reply:
awesome